Exposing Entitlement Management access package assignment approvals in v1.0 (#15132)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/entitlementmanagement-accesspackageassignmentapproval-filterByCurrentUser.md

> ---

Reviewer: Fix file name to be **approval-filterbycurrentuser.md** following the syntax _bindingParameterType-functionName.md_

Reviewer 2: updating

> title: "List accessPackageAssignmentApprovals"
> description: "Get the approval resources from the accessPackageAssignmentApprovals navigation property."
> author: "sbounouh"
> ms.localizationpriority: medium
> ms.prod: "governance"
> doc_type: apiPageType
> ---
> 
> # FilterByCurrentUser accessPackageAssignmentApprovals

Reviewer: Following the guidance, title name has the syntax _bindingParameterType: functionName_.
```suggestion
title: "approval: filterByCurrentUser"
description: "Get the approval resources from the accessPackageAssignmentApprovals navigation property."
author: "markwahl-msft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: apiPageType
---

# approval: filterByCurrentUser
```

Namespace: microsoft.graph

> In Azure AD entitlement management, return a collection of the access package assignment approvals.  With the option of `approver`, this returns the access package assignment approvals that the calling user as an approver is in scope of approving, including those requests they have to take action on.

Reviewer: The `approver` value is required in the URL path so no need to mention it here
```suggestion
In Azure AD entitlement management, return a collection of the access package assignment approvals for which the calling user is an approver.
```

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

| Permission type                        | Permissions (from least to most privileged) |
|:---------------------------------------|:--------------------------------------------|
| Delegated (work or school account)     | EntitlementManagement.ReadWrite.All |
| Delegated (personal Microsoft account) | Not supported. |
| Application                            | Not supported. |

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /identityGovernance/entitlementManagement/accessPackageAssignmentApprovals/filterByCurrentUser(on='approver')
```

> ## Optional query parameters
> This method supports the OData query parameters for paging through a large result set. For general information, see [OData query parameters](/graph/query-parameters).

Reviewer: Functions typically have a function parameters section

Q: approvalFilterByCurrentUserOptions has other values - `target`, `createdBy`, and it seems EM is the only workload using this enum. Does this function also support these two values?
```suggestion
## Function parameters
In the request URL, provide the following query parameters with values.
The following table shows the parameters that can be used with this function.

|Parameter|Type|Description|
|:---|:---|:---|
|on|approvalFilterByCurrentUserOptions| The allowed value is `approver`. Required.|

```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body
Do not supply a request body for this method.

## Response

If successful, this method returns a `200 OK` response code and a collection of [approval](../resources/approval.md) objects in the response body.

## Examples

### Request
<!-- {
  "blockType": "request",

>   "name": "list_approval"

Reviewer: ```suggestion
  "name": "approvalthis-filterbycurrentuser"
```
